frontend_tests/node_tests/hotkey.js

```diff
@@ -341,13 +348,13 @@ run_test('motion_keys', () => {
         }
     }
 
-    function assert_unmapped(name) {
-        assert.equal(process(name), false);
+    function assert_unmapped(name, shiftKey, ctrlKey, metaKey) {
+        assert.equal(process(name, shiftKey, ctrlKey, metaKey), false);
     }
 
-    function assert_mapping(key_name, func_name, shiftKey, ctrlKey) {
+    function assert_mapping(key_name, func_name, shiftKey, ctrlKey, metaKey) {
         stubbing(func_name, function () {
-            assert(process(key_name, shiftKey, ctrlKey));
+            assert(process(key_name, shiftKey, ctrlKey, metaKey));
```

Comment:
Why is this part of this commit?  It seems like we don't use metaKey here.  Let's split it out to its own infrastructure commit adding support for meta keys in the tests (even if we don't use them).